Gruntfile.js

```diff
@@ -110,6 +121,16 @@ module.exports = function (grunt) {
             }
         },
 
+        json_remove_fields: {
+            min_source_maps: {
+                options: {
+                    fields: ['sourceRoot']
```

Comment:
This is a quick fix to get around `grunt-contrib-uglify` limitations. Adding a `sourceRoot` the `uglify` task config is required to get the source contents to actually end up in the map (fixing the issue this PR addresses). But this leads to `"intl-messageformat/intl-messageformat/"` for the source files in the browser dev tools, so this Grunt task removes the extra `"intl-messageformat/"` from the `.min.js.map` files.